app/__init__.py

```diff
@@ -151,6 +152,10 @@ def override_url_for():
     application.config['THEME_PATHS'] = os.path.dirname(os.path.abspath(__file__))
     Themes(application, app_identifier="surveyrunner")
 
+    @application.teardown_appcontext
+    def shutdown_session(exception=None):
```

Comment:
Also I clicked "Ignore" in the GUI to see if it would show a snippet of how to do it; but it didn't (it just hid the issue) and now can't un-ignore it, sorry!